src/Hazelcast.Net/Core/IdleTime.cs

```diff
@@ -12,18 +12,25 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#nullable enable
+
 using System;
 
 namespace Hazelcast.Core
 {
     /// <summary>
-    /// Idle time.
+    /// Provides idle-time constants.
     /// </summary>
-    public static class IdleTime
+    public static partial class IdleTime
     {
         /// <summary>
-        /// A constants used to specify an infinite idle-time (never idles).
+        /// An infinite idle-time (never idle).
+        /// </summary>
+        public static TimeSpan Infinite => TimeSpanExtensions.MinusOneMillisecond;
+
+        /// <summary>
+        /// The default idle-time (use the value configured on the server).
         /// </summary>
-        public static readonly TimeSpan Infinite = TimeSpan.FromMilliseconds(-1);
+        public static TimeSpan Default => TimeSpan.Zero;
```

Comment:
Why do we name this `Default`, but some others as `Zero`?

Reply:
Because... an idle time of zero does not make sense, but for the server passing zero means "use the default, configured value". So `Default` carries that notion.